setup/setup.ui

```diff
@@ -1335,6 +1347,142 @@
                     <property name="position">2</property>
                   </packing>
                 </child>
+                <child>
+                  <object class="GtkFrame" id="frame6">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="label_xalign">0</property>
+                    <property name="shadow_type">none</property>
+                    <child>
+                      <object class="GtkBox" id="vbox11">
+                        <property name="orientation">vertical</property>
+                        <property name="visible">True</property>
+                        <property name="can_focus">False</property>
+                        <property name="spacing">6</property>
+                        <property name="margin_top">6</property>
+                        <property name="margin_start">24</property>
+                        <child>
+                          <object class="GtkCheckButton" id="checkbutton_custom_theme">
+                            <property name="label" translatable="yes">Use custom theme:</property>
```

Comment:
I wish to put GtkCheckButton and GtkComboBox at the same line likes:

"Use custom theme: [Adwaita]"